### PR TITLE
Add error code for ScanOperationStates table

### PR DESCRIPTION
**Pull request by erdembayar**
Addresses: https://github.com/NuGet/Engineering/issues/5250
Adding 2 new columns to persist extra details about scan operation

### CODE REVIEW

File: tests/NuGetGallery.Core.Facts/Frameworks/PackageFrameworkCompatibilityFactoryFacts.cs [hunk 1]
```diff
@@ -217,7 +217,7 @@ public void FrameworksInTableShouldBeOnAscendingOrder()
             foreach (var row in result.Table)
             {
                 var compatibleFrameworks = row.Value;
-                var expectedList = compatibleFrameworks.OrderBy(cf => cf.Framework, new NuGetFrameworkSorter());
+                var expectedList = compatibleFrameworks.OrderBy(cf => cf.Framework, NuGetFrameworkSorter.Instance);
```
Comment (erdembayar) [inline tests/NuGetGallery.Core.Facts/Frameworks/PackageFrameworkCompatibilityFactoryFacts.cs:220]: It happening because one of above updated packages.

**Review by drewgillies — APPROVED** [review]

File: Directory.Build.props [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <NuGetClientPackageVersion>6.9.1</NuGetClientPackageVersion>
-    <ServerCommonPackageVersion>2.117.0</ServerCommonPackageVersion>
-    <NuGetJobsPackageVersion>4.3.0-dev-9116469</NuGetJobsPackageVersion>
+    <ServerCommonPackageVersion>2.120.0-dev-eryondon-extraValidation-9271086</ServerCommonPackageVersion>
```
Comment (joelverhagen) [inline Directory.Build.props:4]: TODO: update to official build

**Review by joelverhagen — DISMISSED** [review]

File: Directory.Build.props [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+    <NuGetJobsPackageVersion>4.3.0-dev-eryondon-extraValid-9271398</NuGetJobsPackageVersion>
```
Comment (lyndaidaii) [inline Directory.Build.props:5]: I think same here, merge PR to dev, and build dev packages instead of personal dev branch.

**Review by lyndaidaii — APPROVED** [review]

File: Directory.Build.props [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <NuGetClientPackageVersion>6.9.1</NuGetClientPackageVersion>
-    <ServerCommonPackageVersion>2.117.0</ServerCommonPackageVersion>
-    <NuGetJobsPackageVersion>4.3.0-dev-9116469</NuGetJobsPackageVersion>
+    <ServerCommonPackageVersion>2.220.0-main-9301227</ServerCommonPackageVersion>
```
Comment (joelverhagen) [inline Directory.Build.props:4]: We typically use stable versions of ServerCommon (for some reason). Probably best to stick with that unless we have a team conversation to deviate from the practice

**Review by lyndaidaii — APPROVED** [review]

**Review by lyndaidaii — APPROVED** [review]

**Pull request merged**